### PR TITLE
Replace CircleCI with GitHub Actions

### PR DESCRIPTION
**Pull request by supl**
This PR replaces CircleCI with GitHub Actions by this opportunity.

After this PR is merged, the CircleCI project can be deleted too
https://app.circleci.com/pipelines/github/scalar-labs/scalardl-javascript-sdk-base

### CODE REVIEW

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

File: .github/workflows/unit-test.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  jest:
```
Comment (kota2and3kan) [inline .github/workflows/unit-test.yml:12]:
```suggestion
  test:
```
Is this typo?

**Review by kota2and3kan — APPROVED** [review]
LGTM! Thank you!

It is not related to funcitions of workflow, but I left a very minor comment.
Please take a look!

File: .github/workflows/unit-test.yml (hunk 1 above)
Comment (supl) [reply .github/workflows/unit-test.yml:12]: Ah! Thank you so much.
It's a mistake.
I copied this from another project and forgot to change this job name. :p

**Pull request merged**